docker-compose-test.yml

```diff
@@ -16,8 +16,11 @@ services:
     - hmpps_int
     container_name: wiremock
     restart: always
+    command: "--global-response-templating --enable-stub-cors" # add command `--verbose` to output verbose request matching
```

Comment:
allows us to grab the request path and use it to find the files for the mock response